yellowbrick/features/base.py

```diff
@@ -189,6 +191,10 @@ def fit(self, X, y=None, **kwargs):
                     str(cdx) for cdx in range(ncols)
                 ]
 
+        # Drop nan-containing rows and warn that this has taken place
+        nan_warnings.warn_if_nans_exist(X)
+        X = nan_warnings.drop_rows_containing_nans(X)
```

Comment:
Should we allow the user to stop this behavior? E.g. have an argument to the visualizer `drop_nans=True`, that we condition on? It feels a little squishy to me to modify the user's dataset, even with a warning; but I'd like to discuss it.

Reply:
Squishy is the perfect word for it! To answer you question, I need to know if I should move the nan removal deeper inside so that the modified data is not returned and still used for plotting. If so, where should I put that? If not, then I agree that the argument is a good plan.

Reply:
Ok. I've thought and played with this and my current preference is to leave the user's data intact and only temporarily clean it in the `RadViz.draw()` method. Now that all the functionality is in place it is trivial to move this around if you disagree or have another suggestion.

Reply:
@Aylr ok that sounds good to me!